css/guideCarouselSwiper.css

```diff
@@ -0,0 +1,11 @@
+.swiper-pagination-bullet {
+  background-color: rgb(239, 130, 0) !important;
```

Comment:
Could we avoid using `!important` here? It might lead to future issues to override bullet styles.